patches/minecraft/net/minecraft/client/model/EntityModel.java.patch

```diff
@@ -0,0 +1,22 @@
+--- a/net/minecraft/client/model/EntityModel.java
++++ b/net/minecraft/client/model/EntityModel.java
+@@ -8,7 +_,8 @@
+ import net.minecraftforge.api.distmarker.OnlyIn;
+ 
+ @OnlyIn(Dist.CLIENT)
+-public abstract class EntityModel<T extends Entity> extends Model {
++public abstract class EntityModel<T extends Entity> extends Model implements net.minecraftforge.client.animation.IAnimatedModel
++{
+    public float f_102608_;
+    public boolean f_102609_;
+    public boolean f_102610_ = true;
+@@ -30,5 +_,9 @@
+       p_102625_.f_102608_ = this.f_102608_;
+       p_102625_.f_102609_ = this.f_102609_;
+       p_102625_.f_102610_ = this.f_102610_;
++   }
++
++   @Override
++   public void gatherAnimatedParts(java.util.Map<String, net.minecraft.client.model.geom.ModelPart> partMap) {
+    }
```

Comment:
Is there a particular reason why this is not a defaulted no-op method in the interface, but instead defined here as a no-op?

Reply:
Just a developer forgetting I could have used a default method. Fixed.